### DISCUSSION TITLE
NextJS + Electron

### DISCUSSION THREAD

**Discussion by aureabelle** [vercel/next.js#17727] [Help]
Hi everyone!

I created a desktop app using NextJS and Electron. Running the app locally works as expected but when packaged, it just gives a blank screen.

How do you run the app when it's packaged? Like doing `npm start` (`electron .`) locally?

I'm a complete newbie with Electron and hoping to get some insights or reference. Thanks!

**Comment by akash07k** [comment]
@aureabelle  have you made any progress on this? I too am struggling with this issue and want to know that how do I exactly bundle and integrate Next.js and Electron
